### PR TITLE
BACKLOG-10540 Move LanguageSwitcher to design-system-kit

### PR DESCRIPTION
**Pull request by sergehuber**
- Added storybook component
- Added notes
- Added tests

Thanks for reviewing this before I create the next PRs in dx-common-webpack, content-manger & content-editor.

### CODE REVIEW

File: packages/design-system-kit/src/components/LanguageSwitcher/LanguageSwitcher.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    lang: PropTypes.string.isRequired,
+    languages: PropTypes.arrayOf(PropTypes.object).isRequired,
+    onSelectLanguage: PropTypes.func.isRequired
+};
+
+LanguageSwitcher.defaultProps = {
+    color: 'default'
+};
+
+export default LanguageSwitcher;
```
Comment (Taybou) [inline packages/design-system-kit/src/components/LanguageSwitcher/LanguageSwitcher.jsx]: Minor: You can add displayName for component debugging.
>LanguageSwitcher.displayName = 'LanguageSwitcher';

File: packages/design-system-kit/src/components/LanguageSwitcher/LanguageSwitcher.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import doc from './LanguageSwitcher.md';
+
+const lang = () => select('Selected language', ['en', 'fr'], LanguageSwitcher.defaultProps.lang);
+const languages = () => [{language: 'en', displayName: 'English'}, {language: 'fr', displayName: 'French'}];
+const color = () => select('Color', ['default', 'inverted'], LanguageSwitcher.defaultProps.color);
+
+storiesOf('Language switcher', module)
+    .addDecorator(withKnobs)
+    .add('One language', () => (
+        <DSProvider><LanguageSwitcher color={color()} lang={lang()} onSelectLanguage={action('language-selected')}/></DSProvider>
```
[the remark this answers is not in the document]
Comment (sergehuber) [reply packages/design-system-kit/src/components/LanguageSwitcher/LanguageSwitcher.stories.jsx]: Indeed I'll correct that.

File: packages/design-system-kit/src/components/LanguageSwitcher/LanguageSwitcher.stories.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+import LanguageSwitcher from './LanguageSwitcher';
+import {DSProvider} from '../../provider';
+import {action} from '@storybook/addon-actions';
+import {select, withKnobs} from '@storybook/addon-knobs';
+import doc from './LanguageSwitcher.md';
+
+const lang = () => select('Selected language', ['en', 'fr'], LanguageSwitcher.defaultProps.lang);
```
[the remark this answers is not in the document]
Comment (sergehuber) [reply packages/design-system-kit/src/components/LanguageSwitcher/LanguageSwitcher.stories.jsx]: Yep good catch, was due to a refactoring I'll correct this.

[the remark this answers is not in the document]
Comment (sergehuber) [reply packages/design-system-kit/src/components/LanguageSwitcher/LanguageSwitcher.stories.jsx]: Fixed

File: packages/design-system-kit/src/components/LanguageSwitcher/LanguageSwitcher.jsx (hunk 1 above)
Comment (sergehuber) [reply packages/design-system-kit/src/components/LanguageSwitcher/LanguageSwitcher.jsx]: done

**Pull request merged**